shopyo/shopyoapi/cmd.py

```diff
@@ -17,6 +17,7 @@
 from shopyoapi.file import trymkdir
 from shopyoapi.file import trymkfile
 from shopyoapi.file import get_folders
+from shopyoapi.file import trycopy
```

Comment:
Import of 'trycopy' is not used. Remove it if it isn't necessary.

Reply:
It is used in L33 and L35